designer/client/footer/footers-edit.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { MouseEvent } from "react";
+import { DataContext } from "../context";
+import randomId from "../randomId";
+import FooterEdit from "./footer-edit";
+
+type Props = {};
+
+type State = {
+  showAddFooter: boolean;
+  footerInput?: any; //TODO: type
+  id?: string;
+};
+
+class FootersEdit extends React.Component<Props, State> {
+  static contextType = DataContext;
+  constructor(props) {
+    super(props);
+    this.state = {
+      showAddFooter: false,
+      footerInput: undefined,
+      id: "",
+    };
+  }
+
```

Comment:
Is this a duplicate of an already existing file in the same folder? they seem to be doing the same thing?